device/config/functions.go

```diff
@@ -0,0 +1,227 @@
+package config
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/xshellinc/tools/constants"
+	"github.com/xshellinc/tools/dialogs"
+	"github.com/xshellinc/tools/lib/help"
+	"github.com/xshellinc/tools/lib/ssh_helper"
+)
+
+// ConfigLocale is a default method to with dialog to configure the locale
+func ConfigLocale(storage map[string]interface{}) error {
+
+	fmt.Println("[+] Default language: ", constants.DefaultLocale)
+	if dialogs.YesNoDialog("Change default language?") {
+		inp := dialogs.GetSingleAnswer("New locale: ", dialogs.EmptyStringValidator, dialogs.CreateValidatorFn(constants.ValidateLocale))
+
+		arr, _ := constants.GetLocale(inp)
+
+		var l string
+		if len(arr) == 1 {
+			l = arr[0]
+		} else {
+			l = arr[dialogs.SelectOneDialog("Please select a locale from a list: ", arr)]
+		}
+
+		storage[GetConstLiteral(Locale)] = strings.Split(l, " ")[0]
+	}
+
+	return nil
+}
+
+// SaveLocale is a default method to save locale into the image
+func SaveLocale(storage map[string]interface{}) error {
+
+	if _, ok := storage[GetConstLiteral(Locale)]; !ok {
+		return nil
+	}
+
+	ssh, ok := storage["ssh"].(ssh_helper.Util)
+	if !ok {
+		return errors.New("Cannot get ssh config")
+	}
+
+	fp := help.AddPathSuffix("unix", constants.MountDir, constants.ISAAX_CONF_DIR, constants.LocaleF)
+	data := fmt.Sprintf(constants.Language+constants.LocaleLang, storage[GetConstLiteral(Locale)], storage[GetConstLiteral(Locale)])
+
+	_, eut, err := ssh.Run(fmt.Sprintf(`echo "%s" > %s`, data, fp))
+	if err != nil {
+		return errors.New(err.Error() + ":" + eut)
+	}
+
+	fp = help.AddPathSuffix("unix", constants.MountDir, constants.ISAAX_CONF_DIR, "environment")
+	data = fmt.Sprintf(constants.LocaleAll, storage[GetConstLiteral(Locale)])
+
+	_, eut, err = ssh.Run(fmt.Sprintf(`echo "%s" > %s`, data, fp))
+	if err != nil {
+		return errors.New(err.Error() + ":" + eut)
+	}
+
+	return nil
+}
+
+// ConfigLocale is a default method to with dialog to configure the keymap
```

Comment:
__[golint]__
* [golint] comment on exported function ConfigKeyboard should be of the form "ConfigKeyboard ..."

<!-- Comment by [SideCI](https://sideci.com) -->